From bb65045fd12aea1f8dbd0d6e18a761726ee53f0f Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Tue, 5 Jul 2022 05:32:24 +0530
Subject: [PATCH] WA- Remove TestHarnessModeService

This change is introduced to fix the
TestHarnessModeService startup failure
and boot Android T.

Change-Id: Ia602c65a014ad21b617541953eeddd4f92e6b421
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Signed-off-by: svenate <salini.venate@intel.com>
---
 services/java/com/android/server/SystemServer.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/java/com/android/server/SystemServer.java b/services/java/com/android/server/SystemServer.java
index ef311c249c5f..0253ff243d0a 100644
--- a/services/java/com/android/server/SystemServer.java
+++ b/services/java/com/android/server/SystemServer.java
@@ -1801,7 +1801,7 @@ public final class SystemServer implements Dumpable {
             }
 
             t.traceBegin("StartTestHarnessMode");
-            mSystemServiceManager.startService(TestHarnessModeService.class);
+            //mSystemServiceManager.startService(TestHarnessModeService.class);
             t.traceEnd();
 
             if (hasPdb || OemLockService.isHalPresent()) {
-- 
2.17.1

